sklearn/tests/test_calibration.py

```diff
@@ -275,6 +275,34 @@ def multiclass_brier(y_true, proba_pred, n_classes):
     assert calibrated_brier < 1.1 * uncalibrated_brier
 
 
+def test_calibration_zero_probability():
```

Comment:
Please add a comment a the beginning of this test to explain its purpose because it's not obvious at first sight

Reply:
I suggested one here: https://github.com/scikit-learn/scikit-learn/pull/19421#discussion_r573736041

Reply:
I added comment.